planner/core/task.go

```diff
@@ -2149,6 +2164,10 @@ func (p *PhysicalHashAgg) attach2TaskForMpp(tasks ...task) task {
 		finalAgg.SetCost(t.cost())
 		return t
 	case MppScalar:
+		prop := &property.PhysicalProperty{TaskTp: property.MppTaskType, ExpectedCnt: math.MaxFloat64, MPPPartitionTp: property.PassThroughType}
+		if !mpp.needEnforceExchanger(prop) {
+			return p.attach2TaskForMpp1Phase(mpp)
```

Comment:
do we need to create MppScalar1Phase and MppScalar2Phase before?

Reply:
I think we can't do it on `exhaustPhysicalPlans`, exhaustPhysicalPlans only explore with property passed by upper operator, but here we need to check the property produced by lower operator.